packages/@expressive-code/plugin-shiki/src/index.ts

```diff
@@ -199,9 +200,11 @@ export function pluginShiki(options: PluginShikiOptions = {}): ExpressiveCodePlu
 								new InlineStyleAnnotation({
 									styleVariantIndex,
 									color: token.color || theme.fg,
+									bgColor: token.bgColor || theme.bg,
 									italic: ((fontStyle & FontStyle.Italic) as FontStyle) === FontStyle.Italic,
 									bold: ((fontStyle & FontStyle.Bold) as FontStyle) === FontStyle.Bold,
 									underline: ((fontStyle & FontStyle.Underline) as FontStyle) === FontStyle.Underline,
+									strikethrough: ((fontStyle & FontStyle.Strikethrough) as FontStyle) === FontStyle.Strikethrough,
```

Comment:
Just to test if everything works, I changed this line...

```diff
-			strikethrough: ((fontStyle & FontStyle.Strikethrough) as FontStyle) === FontStyle.Strikethrough,
+.    strikethrough: true,
```

...and got this output.

![image](https://github.com/user-attachments/assets/b1885959-9a4a-4cd4-b7f5-3d59a087d9f8)

So it should work once we start getting `fontStyle` accurately from Shiki.